packages/panw/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "4.3.0"
```

Since this is a breaking change, let's make this a major version bump.

```suggestion
- version: "5.0.0"
```

---

done!